development.ini

```diff
@@ -36,6 +36,7 @@ use = egg:rutter#urlmap
 
 [composite:indexer]
 use = config:base.ini#indexer
+set local_store = redis
```

Comment:
If this variable is left empty then redis store is bypassed, but the stored state only exists for the life time of the index endpoint request.